src/main/java/br/com/sbk/sbking/core/HandEvaluations.java

```diff
@@ -79,4 +79,66 @@ public boolean isBalanced() {
         && this.getNumberOfDoubletonSuits() <= 1;
   }
 
+  public boolean hasEightOrMoreCardsInAnySuit() {
+    int longestSuit = this.getCardsPerSuit().entrySet().stream().map(Entry::getValue).reduce(0, Math::max);
+    return longestSuit >= 8;
+  }
+
+  public boolean hasSixCardsInAnySuit() {
+    int longestSuit = this.getCardsPerSuit().entrySet().stream().map(Entry::getValue).reduce(0, Math::max);
```

Comment:
This code is repeated and can be easily extracted to a method.

Reply:
I've done something to solve this comment, please evaluate the latest changes.